CMakeLists.txt

```diff
@@ -67,59 +67,19 @@ message(STATUS "Fetching Boost.")
 FetchContent_MakeAvailable(Boost)
 message("Boost sources successfully fetched into ${boost_SOURCE_DIR}")
 
-set(CLP_FFI_JS_BIN_NAME
-    "ClpFfiJs"
-    CACHE STRING
-    "Binary name for the generated .js and .wasm files."
-)
-add_executable(${CLP_FFI_JS_BIN_NAME})
-
-target_compile_features(${CLP_FFI_JS_BIN_NAME} PRIVATE cxx_std_20)
-
 set(CMAKE_EXECUTABLE_SUFFIX ".js" CACHE STRING "Binary type to be generated by Emscripten.")
 
+set(CLP_FFI_JS_ENABLE_EXCEPTION_FLAG
```

Comment:
We have this, `CLP_FFI_JS_EXTRA_LINKER_FLAGS`, and a bunch more linker options in the foreach below. How about defining a single list variable `CLP_FFI_JS_EXTRA_LINK_OPTIONS` (to match `target_link_options`) and just append to it as necessary?

Reply:
If we end up keeping this option, how about `CLP_FFI_JS_ENABLE_EXCEPTIONS_FLAG`? (Plural exceptions.)

Reply:
Also, does this need to be a CACHE variable?

Reply:
My initial thoughts of creating this variable is to make this configurable. Now I think about it again it seems there's probably no reason to change the option to the JS-emulated one (`-fexceptions`) once we decide use one type of exception but not the other. On the other hand, now we have confirmed the effects of enabling `-fwasm-exceptions` are minor, we should probably always enable it given how the CLP core code depends on exceptions for error handling.

I agree that we should create a single list `CLP_FFI_JS_EXTRA_LINK_OPTIONS` and it makes sense for us to inline this exception flag into the list directly.